lib/main.dart

```diff
@@ -40,45 +41,65 @@ class DogList extends StatelessWidget {
   Widget _buildItem(Doggo dog, BuildContext context) {
     return ExpansionTile(
       leading: const Icon(FontAwesomeIcons.paw),
-      title: Text(dog.breeds.primaryBreedName),
+      title: Text(dog.name + ": " + dog.breeds.primaryBreedName),
       children: [
         Padding(
           padding: const EdgeInsets.symmetric(horizontal: 16.0),
           child: Row(
             children: <Widget>[
-              Image.network(
-                dog.media.images.first.url,
-                height: 120,
-              ),
-              Column(
-                children: <Widget>[
-                  Text('Age: ${dog.age}'),
-                  Text('Gender: ${dog.gender}'),
-                  InkWell(
-                    onTap: () async {
-                      Navigator.push(
-                          context,
-                          MaterialPageRoute(
-                            builder: (context) => FullDogView(
-                                  dog: dog,
-                                ),
-                          ));
-                    },
-                    child: Row(
-                      children: <Widget>[
-                        Text('Adopt me!'),
-                        Icon(Icons.launch)
-                      ],
-                    ),
-                  )
-                ],
-              )
+              _dogImage(dog),
+              _dogDescription(dog, context),
             ],
           ),
         ),
       ],
     );
   }
+
+  Widget _dogImage(Doggo dog) {
+    return Padding(
+      padding: const EdgeInsets.all(20.0),
+      child: Container(
+        height: 120.0,
+        child: ClipRRect(
+          borderRadius: BorderRadius.circular(16.0),
+          child: FadeInImage.memoryNetwork(
+            placeholder: kTransparentImage,
```

Comment:
It would be nicer to have an image as placeholder, otherwise the layout seems to be missing something until the image loads. It would also look broken if the request fails. 

I wasn't sure where to get one free of licenses, so I used the `transparent_image` package.

Reply:
cool, yeah I can get us something that's good with licenses. I do this sort of thing a lot. :-)